tests/metrics/estimate_gas.py

```diff
@@ -34,6 +34,7 @@
 
 from nucypher.blockchain.economics import TokenEconomics
 from nucypher.blockchain.eth.agents import NucypherTokenAgent, StakingEscrowAgent, PolicyAgent, AdjudicatorAgent
+from nucypher.crypto.powers import TransactingPower
```

Comment:
Eh? TransactingPower doesn't seem to be used here

Reply:
:heavy_check_mark: